app/assets/stylesheets/components/_tooltip.scss

```diff
@@ -12,6 +12,107 @@
     white-space: normal;
     width: 100%;
   }
+
+  &:focus::before,
+  &:focus::after {
+    opacity: 1;
+    transition-delay: 100ms;
+    visibility: visible;
+  }
+}
+
+.hint--top {
+  &:focus::before {
+    transform: translateY(-8px);
```

Comment:
Yeah I wasn't sure where we stood with having a standard position - I like sticking with just top, I'll cut out the other stuff.